wicket-extensions/src/main/java/org/apache/wicket/extensions/ajax/markup/html/autocomplete/AutoCompleteSettings.java

```diff
@@ -379,4 +381,30 @@ public AutoCompleteSettings setMinInputLength(int minInputLength)
 		this.minInputLength = minInputLength;
 		return this;
 	}
+
+	/**
+	 * Indicates how the Tab key should be handled when having an item in the autocomplete list
+	 * selected.
+	 *
+	 * @return <code>true</code> if the focus should return to the input field, <code>false</code>
+	 * moves the focus to the next component.
+	 */
+	public boolean shouldFocusInputOnTabSelection()

```

Comment:
IMO it would be better to use an enum here instead of boolean.
This way it would be:
1) more clear what the values mean
2) easier to add more values/states later

Reply:
I like the idea!

I will try look into it during the day.

Reply:
I've pushed an update, ut are a bit unsure about the naming